src/celeritas/setup/Problem.cc

```diff
@@ -447,6 +446,13 @@ problem(inp::Problem const& p, ImportData const& imported)
         ActionDiagnostic::make_and_insert(*core_params);
     }
 
+    if (p.diagnostics.status_checker)
+    {
+        // Add detailed debugging of track states
+        // TODO: this environment variable will be replaced by celer::inp soon!
```

Comment:
Which environment variable?

Reply:
Oops stale comment, I originally patched it in with an environment variable but figured it wasn't much more effort to add to the JSON input.